components/Link/Link.unit.test.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import Link from './Link';
+
+const ChildrenExample = 'Unit test';
+
+describe(' Link', () => {
+  it('Should match the default snapshot', () => {
+    const wrapper = mount(
+      <Link href="_blank" rel="noreferrer">
+        Link
+      </Link>,
+    );
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  it('Should match the default snapshot without underline', () => {
+    const wrapper = mount(
+      <Link href="_blank" rel="noreferrer" skin="dark" underline={false}>
+        Link
+      </Link>,
+    );
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  it('Should match the dark snapshot', () => {
+    const wrapper = mount(
+      <Link href="_blank" rel="noreferrer" skin="dark">
+        Link
+      </Link>,
+    );
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  it('Should render the children', () => {
+    const wrapper = mount(
+      <Link href="_blank" rel="noreferrer">
+        {ChildrenExample}
+      </Link>,
+    );
+    expect(wrapper.text()).toEqual(ChildrenExample);
+  });
```

Comment:
@matheusrpp, I missed a test in case the component does not have a child element. Would it make sense to create such a test case?

Reply:
Do you think that is necessary even with children being a required prop?

Reply:
it's ok for me 😎